database/migrations/2025_02_12_121945_create_flights_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('airline_id')->constrained()->cascadeOnDelete();
```

Comment:
Every time is possible. Try to prevent writing 'strings' 

```suggestion
            $table->foreignId(Airline:class)->constrained()->cascadeOnDelete();
```